draft-rswg-rfc7990-updates.mkd

```diff
@@ -96,8 +96,24 @@ and by dropping the requirement that the XML be embedded in the PDF publication
 
 Historical text from {{RFC7990}} such as Section 2 ("Problem Statement"), Section 4 ("Overview of the Decision-Making Process"), and Section 10 ("Transition Plan") are not copied to this document.
 
-Section 7.6 of "RFC Editor Model (Version 3)" {{RFC9280}} says "Once published, RFC Series documents are not changed."
-{{updating}} and {{pub-versions}} in this document update that policy.
+## Changes to RFC 9280 {#changes-to-9280}
+
+Section 7.6 of {{RFC9280}} currently says:
```

Comment:
Is the "currently" anticipaion that we'll change the content as well?

Reply:
Please send such comments to the RSWG mailing list. Thanks!